docs/migration-guide.rst

```diff
@@ -304,7 +304,9 @@ handler, you must call :py:func:`girder.events.unbind` on the existing mapping f
 Async keyword arguments and properties changed to async\_ PR #2817
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-In version 3.7 of python ``async`` is a `reserved keyword argument <https://www.python.org/dev/peps/pep-0492/#deprecation-plans/>`_. To mitigate any issues all instances of ``async`` in the codebase has changed to ``async_``. If the functions are called with ``async`` a deprecation warning will be given. This affects:
+In version 3.7 of python ``async`` is a `reserved keyword argument <https://www.python.org/dev/peps/pep-0492/#deprecation-plans/>`_.
+To mitigate any issues all instances of ``async`` in the codebase has changed to ``asynchronous``.
+If the functions are called with ``async`` a deprecation warning will be given. This affects:
```

Comment:
I realized this last statement is not true. Calling with `async` is now invalid, not deprecated.